.idea/php.xml

```diff
@@ -183,11 +183,14 @@
       <path value="$PROJECT_DIR$/vendor/dealerdirect/phpcodesniffer-composer-installer" />
       <path value="$PROJECT_DIR$/vendor/gewis/gewisphp-coding-standards" />
       <path value="$PROJECT_DIR$/vendor/stripe/stripe-php" />
+      <path value="$PROJECT_DIR$/vendor/symfony/polyfill-php83" />
+      <path value="$PROJECT_DIR$/vendor/symfony/translation-contracts" />
+      <path value="$PROJECT_DIR$/vendor/symfony/validator" />
     </include_path>
   </component>
   <component name="PhpInterpreters">
     <interpreters>
-      <interpreter id="b6502dad-7c78-4cdb-b177-76e9f54dbca6" name="abc-db.docker-registry.gewis.nl/gewisdb_web:development" home="docker://DATA" false="false" debugger_id="php.debugger.XDebug">
+      <interpreter id="b6502dad-7c78-4cdb-b177-76e9f54dbca6" name="abc-db.docker-registry.gewis.nl/gewisdb_web:development" home="docker://DATA" auto="false" debugger_id="php.debugger.XDebug">
```

Comment:
Another open PR does the same changes; is it possible to not include the .idea files?

Reply:
Anyway to now easily remove these files in the pull request?